nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/resources/docs/org.apache.nifi.processors.script.ExecuteScript/additionalDetails.html

```diff
@@ -0,0 +1,698 @@
+    -->
+
+<head>
+    <meta charset="utf-8"/>
+    <title>ExecuteScript</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
+    <style>
+h2 {margin-top: 4em}
+h3 {margin-top: 3em}
+td {text-align: left}
+    </style>
+</head>
+
+<body>
+
+<h1>ExecuteScript</h1>
+
+<h3>Description</h3>
+<p>
+    The ExecuteScript Processor provides the ability to use a scripting language in order to leverage the NiFi API to perform tasks such as the following:
+</p>
+<ul>
+    <li>Read content and/or attributes from an incoming Flowfile</li>
+    <li>Create a new FlowFile (with or without a parent)</li>
+    <li>Write content and/or attributes to an outgoing FlowFile</li>
+    <li>Interact with the ProcessSession to transfer FlowFiles to relationships</li>
+    <li>Read/write to the State Manager to keep track of variables across executions of the processor</li>
+</ul>
+
+<p>
+    <b>Notes:</b>
+<ul>
+    <li>The engine listed as "python" in the list of available script engines is actually Jython, not Python. When using Jython, you cannot import pure (CPython) modules such as pandas</li>
+    <li>Lua does not allow for referencing static members of a class, so the REL_SUCCESS and REL_FAILURE relationships are made available via script bindings (aka variables), see the Variable Bindings section for more details</li>
+    <li>ExecuteScript uses the JSR-223 Script Engine API to evaluate scripts, so the use of idiomatic language structure is sometimes limited. For example, in the case of Groovy, there is a separate ExecuteGroovyScript processor that allows you to do many more idiomatic Groovy tasks. For example it's easier to interact with Controller Services via ExecuteGroovyScript vs. ExecuteScript (see the ExecuteGroovyScript documentation for more details)</li>
+    <li>JavaScript is provided by the Nashorn engine, which is not available in later Java versions, so it may not show up in the Script Engines list depending on the JRE used</li>
+</ul>
+</p>
+<h3>Variable Bindings</h3>
+<p>
+    The Processor expects a user defined script that is evaluated when the processor is triggered. The following variables are available to the scripts:
+</p>
+<table>
+    <tr>
+        <th>Variable Name</th>
+        <th>Description</th>
+        <th>Variable Class</th>
+    </tr>
+    <tr>
+        <td><b>session</b></td>
+        <td>This is a reference to the ProcessSession assigned to the processor. The session allows you to perform operations on FlowFiles such as create(), putAttribute(), and transfer(), as well as read() and write()</td>
+        <td><a href="https://www.javadoc.io/doc/org.apache.nifi/nifi-api/latest/org/apache/nifi/processor/ProcessSession.html">ProcessSession</a></td>
+    </tr>
+    <tr>
+        <td><b>context</b></td>
+        <td>This is a reference to the ProcessContext for the processor. It can be used to retrieve processor properties, relationships, Controller Services, and the StateManager.</td>
```

Comment:
In line 42, it says "State Manager". Should the reference here also include the space?